sdk/ahnlich-client-py/internals/protocol.py

```diff
@@ -0,0 +1,74 @@
+import socket
+
+import config
+from internals import query, server_response
+from internals.exceptions import AhnlichProtocolException
+
+
+class AhnlichProtocol:
+    def __init__(self, address: str, port: int, timeout: int = 5):
+        self.address = address
+        self.port = port
+        self.client = self.connect()
+        self.version = self.get_version()
+        self.timeout = timeout
```

Comment:
Either write some comment or explicitly specify the unit of timeout (ms or seconds)